libs/sdk/skyux-eslint/src/rules/template/no-deprecated-classnames.ts

```diff
@@ -1,6 +1,6 @@
 import { TmplAstTextAttribute } from '@angular-eslint/bundled-angular-compiler';
 import { getTemplateParserServices } from '@angular-eslint/utils';
-import { RuleFix } from '@typescript-eslint/utils/dist/ts-eslint';
```

Comment:
☝️ Removing `dist` from the path is the fix. It now honors the `exports` definition in `@typescript-eslint/utils` package.json.